python/pfs/drp/stella/constructFiberFlatTask.py

```diff
@@ -70,23 +70,29 @@ def combine(self, cache, struct, outputId):
             assert len(value) == 1, "Expect a single answer for this single dataset"
             dithers[value.pop()].append(dataRef)
         self.log.info("Dither values: %s" % (sorted(dithers.keys()),))
-        coadds = {dd: self.combination.run(dithers[dd]) for dd in dithers}
 
         # Sum coadded dithers to fill in the gaps
         sumFlat = None  # Sum of flat-fields
         sumExpect = None  # Sum of what we expect
         for dd in dithers:
-            image = coadds[dd]
-            dataRef = dithers[dd][0]  # Representative dataRef
+            image = self.combination.run(dithers[dd])
+            self.log.info(f"Combined {len(dithers[dd])} images for dither {dd}")
+            # We can have NANs in the combined image, e.g., a pixel that's in the static mask
+            # Can't really interpolate over them, since the traces aren't Nyquist sampled.
+            isBad = np.isnan(image.image.array)
+            if np.any(isBad):
+                image.image.array[isBad] = 0.0
 
+            dataRef = dithers[dd][0]  # Representative dataRef
```

Comment:
There's I/O going on here, because holding the images all in memory at once is not feasible.

I'm looking forward to getting my hands on the Gen3 middleware!